src/types/ChargingStation.tsx

```diff
@@ -140,3 +140,8 @@ export enum ConnectorCurrentLimitSource {
   STATIC_LIMITATION = 'SL',
   CONNECTOR = 'CO'
 }
+
+export interface HttpChargingStationRequest {
```

Comment:
Like in the backend, create a folder named requests and put all Http types in it.